src/prefect/serialization/environment.py

```diff
@@ -40,6 +40,7 @@ class KubernetesJobEnvironmentSchema(ObjectSchema):
     class Meta:
         object_class = KubernetesJobEnvironment
 
+    job_spec_file = fields.String(allow_none=False)
```

Comment:
You probably don't want to do this, since it will trigger an attempted read from the file at deserialization time